### PR TITLE
[FEAT] added a fixed menu bar and some scaling adaptation

### PR DESCRIPTION
**Pull request by vlad-lisnenko**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -22,73 +22,52 @@ html {
...
+  left: 0;
   width: 100%;
-  height: 100%;
-  background-color: #1b1b1b;
-  color: white;
-  display: flex;
-  align-items: flex-start;
-  flex-direction: column;
-  font-family: helvetica-font;
+  transition: background-color 0.3s ease-in-out; /* Добавляем анимацию смены фона */
```
Comment (scratchw0w) [inline css/style.css]: no need in comment

File: index.html [hunk 2]
```diff
@@ -161,14 +163,13 @@ <h3 class="image-description">
       <h2 class="title-for-container-right">V</h2>
```
Comment (scratchw0w) [inline index.html:164]: add spacing

**Review by scratchw0w — APPROVED** [review]

File: css/style.css [hunk 3]
```diff
@@ -22,73 +22,52 @@ html {
   scroll-snap-align: start;
 } */
```
Comment (scratchw0w) [inline css/style.css:24]: please remove body{} and .page comments, I guess we don't need it anymore.

Also could you remove script and other staff linked with smart scrolling.

File: css/style.css [hunk 4]
```diff
@@ -22,73 +22,52 @@ html {
...
 
-/* Main Container */
-
-.container-main {
-  position: relative;
+.scrolling-container {
```
Comment (scratchw0w) [inline css/style.css:25]: Don't we need /* Main Container */ comment in here?

File: css/style.css [hunk 5]
```diff
@@ -201,16 +207,15 @@ mark {
...
   flex-direction: column;
-  align-items: center;
-  margin-left: 100px;
-  margin-top: 40px;
+  margin-left: 10%;
 }
 
 .description {
-  max-width: 300px;
+  /* max-width: 40%; */
```
Comment (scratchw0w) [inline css/style.css:216]: please remove unnecessary comment

File: js/script.js [hunk 6]
```diff
@@ -4,3 +4,16 @@ function scrollToContent(contentId) {
...
     }
 }
+
+
+const scrollingContainer = document.querySelector('.scrolling-container');
+
+window.addEventListener('scroll', () => {
+  const scrollY = window.scrollY;
+
+  if (scrollY > 100) {
```
Comment (scratchw0w) [inline js/script.js:14]: please move magic number to constant

File: js/script.js [hunk 7]
```diff
@@ -4,3 +4,16 @@ function scrollToContent(contentId) {
...
+    scrollingContainer.classList.add('black-bg');
```
Comment (scratchw0w) [inline js/script.js:15]: use const var black-bg instead

File: js/script.js [hunk 8]
```diff
@@ -4,3 +4,16 @@ function scrollToContent(contentId) {
...
+  } else {
+    scrollingContainer.classList.remove('black-bg');
```
Comment (scratchw0w) [inline js/script.js:17]: and here

File: index.html [hunk 9]
```diff
@@ -12,51 +12,53 @@
...
-          class="apply-button"
-          href="javascript:void(0);"
-          onclick="scrollToContent('container-studying')"
-          >apply now</a
-        >
+
+    <div class="scrolling-container">
+      <div class="scrolling-container-size">
+        <div class="icon-and-text">
+          <img class="icon" src="src/computer-logo-border.png" alt="Icon" />
```
Comment (scratchw0w) [inline index.html:21]: I guess there should be alt = Error message, for ex. "Icon loading failed, please try again"

File: index.html [hunk 10]
```diff
@@ -71,7 +73,7 @@ <h1 class="title-left">II</h1>
       <div class="stripe"></div>
 
       <div class="section space-beetwen">
-        <img class="vision right" src="src/vision_2.jpeg" alt="Second Image" />
+        <img class="vision right" src="src/vision_2.jpg" alt="Second Image" />
```
Comment (scratchw0w) [inline index.html:76]: same for alt, error msg

please make it in all alts

File: index.html [hunk 11]
```diff
@@ -12,51 +12,53 @@
...
       </div>
     </div>
-    <!-- </div> -->
+
+    <video class="background-video " autoplay muted loop>
+      <source src="src/background-video.mp4" type="video/mp4" />
+    </video>
 
     <!-- Container Vision -->
     <!-- <div class="page"> -->
```
Comment (scratchw0w) [inline index.html:60]: may be removed

File: index.html [hunk 12]
```diff
@@ -83,6 +85,7 @@ <h1 class="title-left">II</h1>
 
     <!-- Container Team -->
     <!-- <div class="page"> -->
```
Comment (scratchw0w) [inline index.html:87]: may be removed

File: index.html [hunk 13]
```diff
@@ -117,6 +120,7 @@ <h1 class="title-for-container-right">III</h1>
 
     <!-- Container Product -->
     <!-- <div class="page"> -->
```
Comment (scratchw0w) [inline index.html:122]: may be removed

File: index.html [hunk 14]
```diff
@@ -157,18 +159,18 @@ <h3 class="image-description">
 
     <!-- Container-Studying -->
     <!-- <div class="page"> -->
```
Comment (scratchw0w) [inline index.html:161]: may be removed

File: index.html [hunk 15]
```diff
@@ -196,6 +198,7 @@ <h3>Online Course - From Zero to Hero</h3>
 
     <!-- Footer -->
```
Comment (scratchw0w) [inline index.html:199]: may be removed

File: index.html [hunk 16]
```diff
@@ -196,6 +198,7 @@ <h3>Online Course - From Zero to Hero</h3>
...
     <!-- todo: make footer visible through scrolling -->
```
Comment (scratchw0w) [inline index.html:200]: may be removed and script below too

**Review by scratchw0w — CHANGES_REQUESTED** [review]
Please resolve comments

File: css/style.css (hunk 4 above)
Comment (vlad-lisnenko) [reply css/style.css:25]: is this the main container ? its scrollling-container

**Pull request merged**